packages/react-components/src/components/VerticalGallery.tsx

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { DefaultButton, Icon, IStyle, mergeStyles, Stack, Text } from '@fluentui/react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { useTheme } from '../theming';
+import { BaseCustomStyles } from '../types';
+import {
+  childrenContainerStyle,
+  controlBarContainerStyle,
+  leftRightButtonStyles,
+  rootStyle
+} from './styles/VerticalGallery.styles';
+
+export interface VerticalGalleryStyles extends BaseCustomStyles {
+  /** Styles for each video tile in the vertical gallery */
+  children?: IStyle;
+  /** Styles for the previous button in the verticalGallery control bar */
```

Comment:
Does this not style the next button as well?

Reply:
forgot to update the comment from my initial idea